keycloak-rest-api-extensions-tests/src/test/java/io/cloudtrust/keycloak/services/api/admin/CtUserResourceTest.java

```diff
@@ -63,33 +52,33 @@ void sendExecuteActionsEmailUseOtherRealmThemeFailureTest() throws IOException,
     @Test
     void sendExecuteActionsEmailRealmNotFoundTest() throws IOException, URISyntaxException {
         // unknown realm does not exist
-        failureCase("?themeRealm=unknown", 400);
+        failureCase();
     }
 
     private void successCase(String queryParams) throws IOException, URISyntaxException, MessagingException {
-        String id = this.getRealm().users().search("john-doh@localhost").get(0).getId();
+        String id = testRealm.admin().users().search("john-doh@localhost").getFirst().getId();
         String path = String.format(EXECUTE_ACTIONS_EMAIL_FMT, id);
-        int nbReceived = greenMail.getReceivedMessages().length;
+        int nbReceived = mailServer.getReceivedMessages().length;
 
-        this.api().callJSON("PUT", path + queryParams, ACTIONS);
-        assertThat(greenMail.getReceivedMessages().length, is(nbReceived + 1));
-        MimeMessage mail = greenMail.getReceivedMessages()[nbReceived];
+        api().callJSON("PUT", path + queryParams, ACTIONS);
+        assertThat(mailServer.getReceivedMessages().length, is(nbReceived + 1));
+        MimeMessage mail = mailServer.getReceivedMessages()[nbReceived];
         assertThat(mail, is(not(nullValue())));
         String mailContent = getMailContent(mail);
         assertThat(mailContent, containsString("ct-verify-email"));
     }
 
-    private void failureCase(String queryParams, int expectedHttpStatusCode) throws IOException, URISyntaxException {
-        String id = this.getRealm().users().search("john-doh@localhost").get(0).getId();
+    private void failureCase() throws IOException, URISyntaxException {
```

Comment:
Keep the version with parameters even if IntelliJ is telling parameters are always the same